extensions-contrib/tdigestsketch/src/test/java/org/apache/druid/query/aggregation/tdigestsketch/TDigestSketchAggregatorTest.java

```diff
@@ -192,9 +192,9 @@ public void buildingSketchesAtQueryTime() throws Exception
     Object quantilesObject = row.get(1); // "quantiles"
     Assert.assertTrue(quantilesObject instanceof double[]);
     double[] quantiles = (double[]) quantilesObject;
-    Assert.assertEquals(0, quantiles[0], 0.05); // min value
-    Assert.assertEquals(0.5, quantiles[1], 0.05); // median value
-    Assert.assertEquals(1, quantiles[2], 0.05); // max value
+    Assert.assertEquals(0.0, quantiles[0], 0.0); // min value
```

Comment:
[I think travis is failing](https://travis-ci.org/apache/incubator-druid/jobs/626951265#L4545) sql compatible tests because this will produce different values depending on `NullHandling.replaceWithDefault()`. In default mode the nulls are coerced to zeroes, but in sql compatible mode with the changes in this PR, null rows will be ignored.

Reply:
Yep, running the tests with the above JVM param causes tests to fail. How to handle this though? Check for the setting and change the tests to test for aggregated values with rows being ignored?

Reply:
A fairly large portion of the ingestion and query tests have a pattern something like

```Assert.assertEquals(NullHandling.replaceWithDefault() ? x : y, actual)```

to handle differences between the two modes.